requirements.txt

```diff
@@ -3,5 +3,5 @@ first>=2.0.1
 algorithm>=1.2.0
 filesystem_tree>=1.0.1
 dependency_injection>=1.1.0
-aspen>=1.0rc2
+aspen==1.0rc3
```

Comment:
You sure you want `==` instead of `>=`?

Reply:
Yes, this is to ensure that the next release of Pando won't break later like the last one did (see #580).